pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -532,6 +591,12 @@ func (manager *Manager) updateClusterStatus(ctx context.Context, name string, up
 	})
 }
 
+func (manager *Manager) UpdateClusterShardingStatus(ctx context.Context, name string, status *clusterv1alpha2.ClusterStatus) error {
```

Comment:
Changing the parameter to `shardingName` makes it clearer.
`func (manager *Manager) UpdateClusterShardingStatus(ctx context.Context, name string, shardingName *string)`